include/sdf/Box.hh

```diff
@@ -65,6 +68,12 @@ namespace sdf
     /// \return A reference to a gz::math::Boxd object.
     public: gz::math::Boxd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Box
+    /// \param[in] density Density of the box in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the box in kg/m^3
```

Reply:
Corrected in [ac943c6](https://github.com/gazebosim/sdformat/pull/1299/commits/ac943c652f84ff23acc602fd13f00f9d9a32f2a7)